docs/self-managed/concepts/elasticsearch-privileges.md

```diff
@@ -3,32 +3,43 @@ id: elasticsearch-privileges
+If you implement Camunda 8 with Elasticsearch as a service provider, the following [privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html) may be required:
 
 ## Cluster privileges
 
-- `monitor` - necessary for health check
-- `manage_index_templates` to create and manage index schema on start up, if they don't already exist in Elasticsearch.
-- _Optional_ `manage_ilm` - required only when ILM is enabled
+- `monitor` - Required to check the Elasticsearch cluster health. This privilege provides read-only cluster operations permissions.
+- `manage_index_templates` - Creates the necessary index templates when Operate is started for the first time, or when updating to a newer version of Camunda 8. Once the index templates are created, you can stop Operate, remove this privilege, and then start Operate again.
+- `manage_ilm` - _Required when index lifecycle management (ILM) is enabled._ Required to create the necessary ILM policies when Operate is started for the first time, or when updating to a newer version of Camunda 8. Once the ILM policies are created, you can stop Operate, remove this privilege, and then start Operate again.
 
-To use the [backup feature](/self-managed/operational-guides/backup-restore/backup-and-restore.md), you must have snapshot privileges:
+### Backup privileges
 
-- `create_snapshot`
-- `monitor_snapshot`
+To use the [backup feature](/self-managed/operational-guides/backup-restore/backup-and-restore.md), you must have snapshot privileges. You can provide these privileges to Operate before you create a backup, and then revoke them after the backup has been completed:
 
-When [updating](/self-managed/operational-guides/update-guide/introduction.md) to a newer version of Camunda 8 which requires data migration, Operate requires pipelines:
+- `create_snapshot` - Creates a backup, or snapshot, of a running cluster.
+- `monitor_snapshot` - Provides read-only permissions to list and view snapshot details.
 
-- `manage_pipeline`
+### Update privileges
 
-More information on cluster privileges in Elasticsearch can be found [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#privileges-list-cluster).
+When [updating](/self-managed/operational-guides/update-guide/introduction.md) to a newer version of Camunda 8 which requires data migration, Operate requires the following:
+
+- `manage_pipeline` - Allows any data transformations to occur when updating.
+- `manage_index_templates` - See [cluster privileges](#cluster-privileges).
+- `manage_ilm` - _Required when index lifecycle management (ILM) is enabled._ See [cluster privileges](#cluster-privileges).
+
+These privileges can be granted temporarily during an upgrade:
```

Comment:
❓ Just for context, why are we documenting this? If I remember correctly, I summarized that in the context of a customer request.

Reply:
I don't have context for the original ask, only the [summary](https://github.com/camunda/camunda-docs/issues/3355#issue-2144900948). Can you specify if these permissions should be removed, and if so, which ones in particular?